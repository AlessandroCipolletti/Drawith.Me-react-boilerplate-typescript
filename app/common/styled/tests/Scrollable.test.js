import React from 'react'
import { render } from 'react-testing-library'

// Import de Theme Provider
import { ThemeProvider } from 'styled-components'
import Theme from 'common/Theme'

import {
  Scrollable,
  ScrollableCss,
} from '../Scrollable'

describe('<Scrollable />', () => {
  const renderComponent = (props = {}) =>
    render(
      <ThemeProvider theme={Theme}>
        <Scrollable {...props}>
          Content
        </Scrollable>
      </ThemeProvider>,
    )

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = renderComponent()

    expect(firstChild).toMatchSnapshot()
  })

  it('Should have public export ScrollableCss', () => {
    expect(ScrollableCss).toBeDefined()
  })
})
