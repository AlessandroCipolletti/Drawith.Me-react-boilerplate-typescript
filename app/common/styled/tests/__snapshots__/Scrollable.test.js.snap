// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Scrollable /> Should render and match the snapshot 1`] = `
<div
  class="Scrollable-w0eb6g-0 hruisM"
>
  Content
</div>
`;
